src/crypto/tls/handshake_client_tls13.go

```diff
@@ -245,6 +284,45 @@ func (hs *clientHandshakeStateTLS13) processHelloRetryRequest() error {
 		hs.transcriptInner.Reset()
 		hs.transcriptInner.Write([]byte{typeMessageHash, 0, 0, uint8(len(chHash))})
 		hs.transcriptInner.Write(chHash)
+
+		// Check for ECH acceptance confirmation.
+		if len(hs.serverHello.ech) == 8 {
+			echAcceptConfHRRTranscript := cloneHash(hs.transcriptInner, hs.suite.hash)
+			if echAcceptConfHRRTranscript == nil {
+				c.sendAlert(alertInternalError)
+				return errors.New("tls: internal error: failed to clone hash")
+			}
+
+			signal := hs.serverHello.ech
+			hs.serverHello.ech = []byte{0, 0, 0, 0, 0, 0, 0, 0}
+			hs.serverHello.raw = nil
+			echAcceptConfHRRTranscript.Write(hs.serverHello.marshal())
+			hs.serverHello.ech = signal
+			hs.serverHello.raw = nil
+
+			echAcceptConfHRR := hs.suite.expandLabel(
+				hs.suite.extract(hs.helloInner.random, nil),
+				echAcceptConfHRRLabel,
+				echAcceptConfHRRTranscript.Sum(nil),
+				8)
+
+			// If this is a test, then assert the client and server have
+			// computed the acceptance signal correctly.
+			//
+			// TODO(cjpatton): Remove this assertion and use the acceptance
```

Comment:
Is this comment up to date? (It seems to reference some old code before -11)

Reply:
This comment is up to do date. We won't refactor the client state machine until #91.